src/mainboard/protectli/vault_adl_n/variants/vp2430/overridetree.cb

```diff
@@ -0,0 +1,71 @@
+chip soc/intel/alderlake
+	# FSP configuration
+
+	# SATA
+	register "sata_ports_enable[0]" = "1" # Header
+	register "sata_ports_enable[1]" = "1" # M.2 2280
+	register "sata_ports_dev_slp[0]" = "1"
+	register "sata_ports_dev_slp[1]" = "1"
+
+	device domain 0 on
+		device ref igpu on
+			subsystemid 0x8086 0x2212
+			register "ddi_portA_config" = "1" # HDMI on port A
+			register "ddi_portB_config" = "1" # DP on port B
+			register "ddi_ports_config" = "{
+				[DDI_PORT_A] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_B] = DDI_ENABLE_HPD | DDI_ENABLE_DDC,
+				[DDI_PORT_1] = DDI_ENABLE_HPD,
+				[DDI_PORT_2] = DDI_ENABLE_HPD,
+			}"
+		end
+
+		# SATA
+		device ref pcie_rp11 off end
+		device ref pcie_rp12 off end
+		# LAN1
+		device ref pcie_rp2 on
+			register "pch_pcie_rp[PCH_RP(2)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_AER | PCIE_RP_CLK_REQ_UNUSED,
+				.pcie_rp_aspm = ASPM_DISABLE,
+				.PcieRpL1Substates = L1_SS_DISABLED,
+				.clk_src = 0,
+				.clk_req = 0,
+			}"
+		end
+		# LAN2
+		device ref pcie_rp3 on
+			register "pch_pcie_rp[PCH_RP(3)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_AER | PCIE_RP_CLK_REQ_UNUSED,
+				.pcie_rp_aspm = ASPM_DISABLE,
+				.PcieRpL1Substates = L1_SS_DISABLED,
+				.clk_src = 1,
+				.clk_req = 1,
+			}"
+		end
+		# LAN 3
+		device ref pcie_rp4 on
+			register "pch_pcie_rp[PCH_RP(4)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_AER | PCIE_RP_CLK_REQ_UNUSED,
+				.pcie_rp_aspm = ASPM_DISABLE,
+				.PcieRpL1Substates = L1_SS_DISABLED,
+				.clk_src = 2,
+				.clk_req = 2,
+			}"
+		end
+		# LAN4
+		device ref pcie_rp7 on
+			register "pch_pcie_rp[PCH_RP(7)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_AER | PCIE_RP_CLK_REQ_UNUSED,
+				.pcie_rp_aspm = ASPM_DISABLE,
+				.PcieRpL1Substates = L1_SS_DISABLED,
+				.clk_src = 3,
+				.clk_req = 3,
+			}"
+		end
+
```

Comment:
There is no NVME device here, if it is in the baseboard devicetree, then it should be moved to override tree. The SMBISO slot description will be invalid, because VP2430 has only x2 link

Reply:
Wasn't sure since in the schems it says `M.2_NVME 2X` on page 6, and `M2_NVME4X` on page 11 :thinking: [Moved](PCIE3_P0_CLK_DN) to `overridetree` as `2X`

Reply:
`M2_NVME4X` is the name of the slot, because it can hold u to 4 PCIe links. However only two of them are connected...

Reply:
BTW the link to the commit is invalid

Reply:
Ayes, that explains it, thanks. [Correct](https://github.com/Dasharo/coreboot/pull/576/commits/734785c0a950d3048c11ac982335b31262decdca#diff-c6b9e16af2e91a2485033bd2f95d2ab30869f1b652824cdb9eca037d0826bfd2R69) link